controlplane/client/client.go

```diff
@@ -138,7 +138,7 @@ func (c *Client) ConfigureSampler(ctx context.Context, samplerResource, samplerN
 
 	status := samplerConfRes.SamplerConfRes.GetStatus()
 	if status.GetType() != protos.Status_OK {
-		return fmt.Errorf("error configuring rule rules: %s", status.GetErrorMessage())
+		return fmt.Errorf("%s", status.GetErrorMessage())
```

Comment:
similarly, in the previous method we have the prefix `error getting samplers list`, why in this case we should not have it, and if so, if there isn't any good reason it should be consistent with the other errors.

Reply:
`error configuring rule rules:` prefix was always returned. When it was firstly introduced, the oly thing it could be created where streams, and the only possible error a rule error. But now we have multiple types of errors and primitives to be configured and the error does not make sense. I could change it to another one, but it will need to be really general, I think it's better that the error function contains all the error information (it will know the context and why it happened)